app/views/miq_ae_class/_instance_fields.html.haml

```diff
@@ -17,7 +17,8 @@
         %th= _('Message')
       %tbody
         - @ae_class.ae_fields.each do |record|
-          - ae_value = record.ae_values.find_or_initialize_by_instance_id(@record)
+          - ae_value = record.ae_values.where("instance_id = ?", @record.id).first
+          - ae_value ||= MiqAeValue.new(:instance_id => @record.id, :field_id => record.id)
```

Comment:
Also, the inner `record` vs the outer `@record` is confusing since the name is *almost* the same.